scripts/rebuild_for_camera.sh

```diff
@@ -0,0 +1,27 @@
+#This script can be used to rebuild the SDK depending on various platforms and cameras.
+#Its purpose is to be called from a shortcut placed on desktop.
+#It expects the parameters passed as the CMAKE command.
+
+#Possible command for the camera: -DUSE_FXTOF1=1 
+#Possible commands for the board: -DDRAGONBOARD=1 -DRASPBERRYPI=1 -DJETSON=1 -DXAVIERNX=1 -DXAVIER=1
+#The implicit build, if no parameters are passed is for AD-96TOF1-EBZ build on linux.
+
+#!/bin/bash
```

Comment:
The shebang should be on the first line of the script in order to have any effect.
https://stackoverflow.com/questions/12910744/why-should-the-shebang-line-always-be-the-first-line